descwl_shear_sims/sim.py

```diff
@@ -83,6 +83,7 @@ def make_sim(
     coadd_dim=None,
     simple_coadd_bbox=False,
     draw_noise=True,
+    im_precision="float",
```

Comment:
I think we should call this im_dtype and accept numpy dtype strings. That will eliminate some complexity here.